apps/editors/tests/test_models.py

```diff
@@ -471,6 +479,23 @@ def test_award_points(self):
         eq_(ReviewerScore.objects.all()[0].score,
             amo.REVIEWED_SCORES[amo.REVIEWED_ADDON_FULL])
 
+    def test_award_points_bonus(self):
+        user2 = UserProfile.objects.get(email='admin@mozilla.com')
+        addon_objects = create_addon_file(
+            u'AwardBonus',
+            u'1.0',
+            amo.STATUS_NOMINATED,
+            amo.STATUS_UNREVIEWED,
+            nomination=(datetime.now() - timedelta(days=11))
+        )
+        self._give_points(user2, addon_objects['addon'], 1)
+        score = ReviewerScore.objects.all()[0]
+        eq_(
+            score.score,
+            amo.REVIEWED_SCORES[amo.REVIEWED_ADDON_PRELIM] +
+            REVIEWED_OVERDUE_BONUS
+        )
```

Comment:
Since we switched to pytest, we use simple `assert`s:
```assert score.score == ...```

Here, you're only testing that a review that has been waiting for longer than the threshold gets a bonus, but not that this bonus is for each day over the threshold. Could you modify this test to instead check the bonus for multiple days over the threshold?